aaad/authenticators.py

```diff
@@ -1,13 +1,25 @@
-from passlib.apache import HtpasswdFile
+
 import os
 
-htpasswd_file = os.environ['HTPASSWD_FILE']
+from passlib.apache import HtpasswdFile
+
+
+htpasswd_files = os.environ['HTPASSWD_FILES']
 
 class FileAuthenticator:
     class __FileAuthenticator:
-        def __init__(self, filename):
-            self.filename = filename
-            self.data = HtpasswdFile(self.filename)
+        def __init__(self, filenames_spec):
+            self.data = self._parse_htpasswd_files(filenames_spec)
+
+        def _parse_htpasswd_files(self, filenames_spec):
+            data = ""
+            filenames = filenames_spec.split(',')
+            for item in filenames:
+                filename = item.strip()
+                if filename:
+                    with open(filename, 'r') as data_file:
+                        data = data + data_file.read()
```

Comment:
Ideally, if these string datum read from different files would be read in a list and then the concatenation happened using a join() only at the last after all the files are read, it would be most efficient, but in this case, might not be a big deal.